docs/source/releases.rst

```diff
@@ -2,6 +2,19 @@ Release History
 ###############
 
 
+v1.5.6 (2024-08-20)
+===================
+
+Maintenance
+-----------
+- Updates test suite and some plans for numpy 2.0 compatibility (primarily with updated bluesky >1.12.0)
```

Comment:
I think we shouldn't release a tag that says 2.0 compatibility and then pins <2.0
Maybe let's just not release nabs right now?

Reply:
har you're right.  I thought we needed to re-release nabs for https://github.com/pcdshub/nabs/issues/81 ? or are you thinking of re-making a tag at the same point in the commit history?

Reply:
Agh I'm getting pinned by my own comments

Reply:
Maybe I'll just do a manual upload of v1.5.5 and we can table this module for a bit?

Reply:
Alternatively: switching `round_` to `round` isn't a very big lift, which is what we were erroring on.

Reply:
Nevermind, the actual issue is in `dask`

Reply:
I'm ok either way, if manually uploading 1.5.5 isn't hard maybe that's the simpler solution

Reply:
I'll go ahead and manually upload 1.5.5 right now as soon as I remind myself how

Reply:
I did eventually figure this out via reading the ci helpers code

Reply:
Can we modify the notes to not mislead us later? I think that's enough to merge.
```suggestion
- Updates test suite and some plans for numpy 2.0 compatibility (primarily with updated bluesky >1.12.0).
  Note that the dependencies of nabs are still not numpy 2.0 compatible.
```